etna/pipeline/autoregressive_pipeline.py

```diff
@@ -173,6 +176,7 @@ def _forecast(self, ts: TSDataset, return_components: bool) -> TSDataset:
 
         return prediction_ts
 
+    # TODO: зачем это надо? Мб удалить?
```

Comment:
We should discuss this.

Reply:
It seems that everything okey here, what do you want to discuss?

Reply:
I can't wee why do we need this thing. It seems to me it does nothing.